tests/unit/AmalgamatingBusinesses.spec.ts

```diff
@@ -24,7 +27,8 @@ describe('Amalgamating Businesses - components and validity', () => {
 
   it('renders as staff', async () => {
     // set state
-    await store.setKeycloakRoles(['staff'])
+    store.setKeycloakRoles(['staff'])
+    await Vue.nextTick()
```

Comment:
Awaiting the store action worked but VS Code said "await has no effect on the type of this expression" which is what you'd see if setKeycloakRoles wasn't async.

So I did this instead, which also works and doesn't have a VS Code message that someone might want to optimize (remove) and thus break the test.